.github/workflows/BuildInfrastructure.yml

```
@@ -0,0 +1,43 @@
+﻿name: Build infrastructure
+
+on:
+  workflow_call:
+    secrets:
+      AZURE_CLIENT_ID:
+        required: true
+      AZURE_TENANT_ID:
+        required: true
+      AZURE_SUBSCRIPTION_ID:
+        required: true
+    inputs:
+      environment:
+        required: true
+        type: string
+
+jobs:
+  build:
+    name: Build infrastructure for ${{ inputs.environment }}
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    environment: ${{ inputs.environment }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: OIDC Login to Azure Public Cloud with AzPowershell
```

```suggestion
      - name: OIDC Login to Azure Public Cloud
```

Skulle vært fjerna fra den andre plassen vi gjør det og, henger igjen fra en tidligere versjon 🙈

---

ah yeah stussa litt på den